lib/dfe/reference_data/itt/cycles.rb

```diff
@@ -0,0 +1,111 @@
+require 'tzinfo'
+require 'time'
+require 'date'
+
+module DfE
+  module ReferenceData
+    module ITT
+      CYCLES_SCHEMA = {
+        id: :integer,
+        find_opens: :datetime,
+        apply_opens: :datetime,
+        show_deadline_banner: { kind: :optional, schema: :datetime },
+        show_summer_recruitment_banner: { kind: :optional, schema: :datetime },
+        apply_1_deadline: :datetime,
+        apply_2_deadline: :datetime,
+        end_of_cycle: :datetime,
+        find_closes: :datetime,
+        holidays: { kind: :map, key: :symbol, value: :daterange }
+      }.freeze
+
+      # Timezone used to specify dates+times in this file
+      TIMEZONE = TZInfo::Timezone.get('Europe/London')
+
+      def self.make_local_time(*args)
+        TIMEZONE.local_to_utc(DateTime.new(*args))
+      end
+
+      CYCLES = DfE::ReferenceData::HardcodedReferenceList.new(
+        {
+          2019 => {
+            find_opens: make_local_time(2018, 10, 6, 9),
+            apply_opens: make_local_time(2018, 10, 13, 9),
+            apply_1_deadline: make_local_time(2019, 8, 24, 18),
+            apply_2_deadline: make_local_time(2019, 9, 18, 18),
+            end_of_cycle: make_local_time(2021, 9, 29, 23, 59, 59),
+            find_closes: make_local_time(2019, 10, 3, 23, 59, 59),
+            holidays: {}
+          },
+          2020 => {
+            find_opens: make_local_time(2019, 10, 6, 9),
+            apply_opens: make_local_time(2019, 10, 13, 9),
+            show_deadline_banner: make_local_time(2020, 8, 1, 9),
+            show_summer_recruitment_banner: make_local_time(2020, 7, 1, 9),
```

Comment:
I’d be tempted to drop these, as they're really only used for Apply and aren't defined externally. (Also subject to change if we decide to show the banners earlier or later)

```suggestion
```

Reply:
Done